internal/cli/log_streams_event_bridge.go

```diff
@@ -97,7 +97,7 @@ func updateLogStreamsAmazonEventBridgeCmd(cli *cli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "eventbridge",
-		Args:  cobra.NoArgs,
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
This is a bug where we weren't accepting any log stream id for the update command. We fix this here and in other update commands down below.